scss/_utilities_off-screen.scss

```diff
@@ -3,7 +3,6 @@
 // Positions element out of flow & off screen for screen readers
 @mixin vf-u-off-screen {
   .u-off-screen {
-    height: 1px !important;
```

Comment:
Maybe safer to make it `height: 0 !important` to make sure it doesn't get any height from content put into it?